doc/news/_preparation_next_release.md

```diff
@@ -34,11 +34,34 @@ docker run -it elektra/elektra
 ## Highlights
 
 - `kdb` now prohibits write operations on cascading keys that miss a corresponding existing key. See the details in the `Tools` section below and the new subsection on cascading writes in the [tutorial](/doc/tutorials/cascading.md) on cascading keys for further information. _(Alexander Firbas)_
-- <<HIGHLIGHT1>>
 - <<HIGHLIGHT2>>
 - <<HIGHLIGHT3>>
 
-### <<HIGHLIGHT1>>
+### Redshift and Elektra
+
+We have created a version of [Redshift](https://github.com/jonls/redshift/) that uses Elektra for configuration management!
+
+Redshift's code for loading configuration files, parsing CLI options and validating configuration was refactored to use Elektra.
+
+To test it, clone the [elektrify branch](https://github.com/qwepoizt/redshift/tree/improve-config/elektrify) and follow the instructions provided in [CONTRIBUTING.md](https://github.com/qwepoizt/redshift/blob/improve-config/elektrify/CONTRIBUTING.md).
```

Comment:
Sorry, I don't understand. Are you proposing we link: 
1. to redshift's PR 837 instead of the elektrify branch and 
2. to https://github.com/jonls/redshift/blob/632fb69c0d7c60b9b18ec87430e54e3f582c8eff/CONTRIBUTING.md instead of https://github.com/qwepoizt/redshift/blob/improve-config/elektrify/CONTRIBUTING.md
?

Reply:
> 1. to redshift's PR 837 instead of the elektrify branch

Yes, to the PR 837, so that people leave comments at the right place.

Reply:
Good idea, thanks, I've updated that accordingly.